package.json

```diff
@@ -2,15 +2,15 @@
   "private": true,
   "scripts": {
     "build": "turbo run build",
-    "dev": "dotenv -- turbo run dev",
+    "dev": "turbo run dev",
     "lint": "turbo run lint",
     "format": "prettier --write \"**/*.{ts,tsx,md}\""
   },
   "devDependencies": {
     "eslint": "^7.32.0",
     "eslint-config-custom": "workspace:*",
     "prettier": "^2.5.1",
-    "turbo": "latest"
```

Comment:
I think being this relaxed on versions is going to hurt eventually. Releases of `turbo` seem to be kinda reasonable.